src/Cli/dotnet/SudoEnvironmentDirectoryOverride.cs

```diff
@@ -33,7 +37,57 @@ public static void OverrideEnvironmentVariableToTmp(ParseResult parseResult)
             if (!OperatingSystem.IsWindows() && IsRunningUnderSudo() && IsRunningWorkloadCommand(parseResult))
             {
                 Directory.CreateDirectory(SudoHomeDirectory);
+
+                var homeBeforeOverride = Path.Combine(Environment.GetEnvironmentVariable("HOME"));
                 Environment.SetEnvironmentVariable("HOME", SudoHomeDirectory);
+
+                CopyUserNuGetConfigToOverridenHome(homeBeforeOverride);
```

Comment:
The spelling is "[overridden](https://en.wiktionary.org/wiki/overridden#English)".